### PR TITLE
limit the build status badge to the main branch

### PR DESCRIPTION
**Pull request by nickpelton**
## What does this do/fix?

The current build status badge in the readme will show failures on all branches, it should be limited to the main branch only.

Docs: https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge#using-the-branch-parameter

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,4 @@ SquareOne is a development framework lovingly maintained by Modern Tribe used fo
 ## Support
 
 Usage of SquareOne is not actively supported by Modern Tribe outside of client contracts. Pull requests and suggestions are welcome and will be addressed based on business need.
+feature/
```
Comment (Vinsanity) [inline README.md:22]: this was probably an accident, yah?

**Review by Vinsanity — APPROVED** [review]
Just one thing to review. 
🧤 🎠

File: README.md (hunk 1 above)
Comment (nickpelton) [inline README.md:22]:
```suggestion
```

Comment (nickpelton) [reply README.md]: Yup, good catch.

**Review by dpellenwood — APPROVED** [review]
:shipit:

**Pull request merged**